mmcls/models/heads/arcface_head.py

```diff
@@ -157,15 +266,16 @@ def loss(self, feats: Tuple[torch.Tensor],
         Returns:
             dict[str, Tensor]: a dictionary of loss components
         """
-
+        label_target = torch.cat([i.gt_label.label for i in data_samples])
+        # Unpack data samples and pack targets
         if 'score' in data_samples[0].gt_label:
             # Batch augmentation may convert labels to one-hot format scores.
             target = torch.stack([i.gt_label.score for i in data_samples])
         else:
-            target = torch.cat([i.gt_label.label for i in data_samples])
+            target = label_target
 
-        # The part can be traced by torch.fx
-        cls_score = self(feats, target)
+        # the index format target would be used
+        cls_score = self(feats, label_target)
```

Comment:
Implementation during mixup may not be correct.

I don't know which implementation is better, what do you think?
https://github.com/cavalleria/cavaface/blob/master/train.py
https://www.kaggle.com/competitions/happy-whale-and-dolphin/discussion/319941

Reply:
I think the first one, mixing the data, is more reasonable. Mixing the embedding seems weird. Just the embedding of the mixing, which I think only modifies the weight of the final average loss.

Support batch-augments is a little difficult in mmcls. Seem we can not get the specific target index and prem index in the head.

Reply:
Now, Arcface should have supported train with batch-augments.  @okotaku

Reply:
It looks good.

Reply:
After discussion, considering to release a new version soon. We decided that in this PR, **only index-format targets are supported**, and for the batch-augment solution, we need to do some experiments and then determine.

The [first solution](https://github.com/cavalleria/cavaface/blob/master/train.py)  does not take into account the changes to logit by mix-target-label in https://github.com/cavalleria/cavaface/blob/822651f0e6d4d08df5441922acead39dc5375103/train.py#L261. The second one is just mix-embedding.